<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="THB-ARG-455.local" ignored="0" name="Test" tests="2" failures="1" timestamp="2023-10-01T11:10:11 EET" time="122.197" errors="0">
  <testcase name="runScenario" time="76.587" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="44.985" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at stepDefinitions.TestNews.check_the_links_within_the_news_content(TestNews.java:50)
at âœ½.Check the links within the news content(file:///Users/maltunbas/git/CommencisMuhammetAltunbas/src/test/java/features/LatestNews.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Test -->
